docs/data-tunnel/07-ibc-intergation.md

```diff
@@ -0,0 +1,100 @@
+# IBC Integration
```

Comment:
can we create a new topic e.g. route types, and then group these route integrations being under it?